tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -113,6 +116,12 @@ AGENT_INIT          When set to "yes", use ${AGENT_BIN} as init process in place
                     of systemd.
                     Default value: no
 
+AGENT_POLICY_FILE   Path to the agent policy rego file to be set in the rootfs.
+                    If defined, this overwrites the default setting of the
+                    permissive policy file. The path is relative to the script
+                    directory designated by the script_dir variable.
+                    Default value: ../../../src/kata-opa/allow-all.rego
```

Comment:
../../../ looks kinda ugly in the help text. Unless you have better ideas to beautify this, I would:
- Document that files must be present in src/kata-opa/, and
- Let the user customize just the file name, not the path.